test/unit/helpers/util/Expect.js

```diff
@@ -1,4 +1,4 @@
-/* eslint max-statements:[2, 28] */
+/* eslint max-statements:[2, 30] */
```

Comment:
can we revert this? It is probably not needed anymore

Reply:
Done